### PR TITLE
Update showcase.json

### PR DESCRIPTION
**Pull request by scott4044**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: showcase.json [hunk 1]
```diff
@@ -3246,6 +3246,30 @@
   "logo": "/img/showcase/zondax-logo.jpg",
   "screenshots": ["/img/showcase/vault-k8s.jpg"],
   "submittableId": "40715473"
- }
+ },
+    {
+  "name": "BitBasel Art Marketplace",
+  "description": "A curated marketplace for buying and selling digital artwork. Built on the Internet Computer leveraging the Bitcoin Ordinals Protocol",
+  "website": "https://bitbasel.com/",
+  "logo": "/img/showcase/bitbasel-art-marketplace_logo.webp",
```
Comment (jennifertrin) [inline showcase.json:3254]: Can you please include the logo in the correct path in the PR?

File: showcase.json [hunk 2]
```diff
@@ -3246,6 +3246,30 @@
...
+  "screenshots": [
```
Comment (jennifertrin) [inline showcase.json:3255]: Can you please include the screenshot in the correct path in the PR?

**Review by jennifertrin — CHANGES_REQUESTED** [review]

**Comment by jennifertrin** [comment]
@scott4044 Can you please review the comments and resolve them above?

**Comment by scott4044** [comment]
@jennifertrin I believe i resolved everything!

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**